dora/core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -61,6 +63,32 @@ public URIStatus(FileInfo info) {
   public URIStatus(FileInfo info, @Nullable CacheContext context) {
     mInfo = Preconditions.checkNotNull(info, "info");
     mCacheContext = context;
+    mIsFromUFS = false;
+  }
+
+  /**
+   * @return if the FileInfo is from UFS instead of worker
+   *
+   * If true, it means this status is retrieved by falling back to UFS.
+   */
+  public boolean getIsFromUFS() {
```

Comment:
prefer the name `IsFromUfs()` for this getter method.

Reply:
OK